Fix : Removed the case status field from the case management tab

samyuktha31 opened the pull request:
Removed the case status field from the case management tab.

Attached the script that needs to be run before publishing the database.

[Script.txt](https://github.com/rakeshrohan/MyChoices/files/1343777/Script.txt)

rakeshrohan commented:
@samyuktha31 - why didnt you just alter table instead of creating a temp table and copying the contents?

samyuktha31 commented:
Changed the script by just altering the table -
[Script.txt](https://github.com/rakeshrohan/MyChoices/files/1362297/Script.txt)

I remember I tried it before and it didn't work . Changed the script now and it works fine.
